tasks/runtime.yml

```diff
@@ -43,6 +43,11 @@
   notify:
     - Restart Munge service
 
+- name: Set fact for login-only nodes
+  # needed b/c openhpc_enable doesn't end up in hostvars
+  set_fact:
+    openhpc_login_only: "{{ openhpc_enable.login | default(false) | bool }}"
```

Comment:
Should this be something like: `openhpc_login_only: "{{ openhpc_enable.login | default(false) | bool and not  openhpc_enable.batch | default(false) | bool }}"`. So we don't get the extra templating in slurm.conf for the combined case?